pkg/processor/trigger/http/trigger.go

```diff
@@ -225,6 +245,24 @@ func (h *http) AllocateWorkerAndSubmitEvent(ctx *fasthttp.RequestCtx,
 	return response, false, nil, processError
 }
 
+func (h *http) allocateWorker(timeout time.Duration) (*worker.Worker, int, error) {
+	// allocate a worker
+	workerInstance, err := h.WorkerAllocator.Allocate(timeout)
+	if err != nil {
+		h.UpdateStatistics(false, 1)
+		return nil, -1, errors.Wrap(err, "Failed to allocate worker")
+	}
+
+	// use the event @ the worker index
+	// TODO: event already used?
```

Comment:
I know it is not you TODO, but do you have an idea what does it mean?

Reply:
@TomerShor No, but I can assume that it is only relevant for normal flow with single event processing, because in that case we use pre-defined events to wrap the context